pages.es/common/chown.md

```diff
@@ -1,24 +1,28 @@
 # chown
 
-> Cambia la propiedad de usuario y grupo sobre archivos y directorios.
+> Cambia la propiedad de usuario y grupo de archivos y directorios.
 > Más información: <https://www.gnu.org/software/coreutils/chown>.
 
 - Cambia el usuario propietario de un archivo/directorio:
 
-`chown {{usuario}} {{ruta/hacia/archivo_o_directorio}}`
+`chown {{usuario}} {{ruta/al/archivo_o_directorio}}`
 
 - Cambia el usuario y grupo propietario de un archivo/directorio:
 
-`chown {{usuario}}:{{grupo}} {{ruta/hacia/archivo_o_directorio}}`
+`chown {{usuario}}:{{grupo}} {{ruta/al/archivo_o_directorio}}`
 
-- Cambia de forma recursiva el propietario sobre un directorio y su contenido:
+- Cambia el usuario propietario y el grupo para que ambos tengan el nombre `user`:
```

Comment:
I guess the ownership concept is better suited with

Cambia el usuario propietario y el grupo para que ambos sean `usuario`